packages/web-components/src/components/rux-timeline/rux-track/rux-track.tsx

```diff
@@ -63,7 +63,7 @@ export class RuxTrack {
         this._handleSlotChange = this._handleSlotChange.bind(this)
     }
 
-    calculateGridColumnFromTime(time: any) {
+    private _calculateGridColumnFromTime(time: any) {
```

Comment:
Would it be possible to document these methods? I wouldn’t necessarily know what makes them private or public at a glance.

Reply:
Absolutely. Lmk what questions you still have. By default, every method is private. You'll know if something is public if you see Stencils Method decorator and documentation. We have very few public methods. 

Also, a nit on Stencil is that all public methods need to be async.

Reply:
Thank you!